airdrop/job/reward_processors/nunet_reward_processor.py

```diff
@@ -116,6 +135,7 @@ def process_rewards(self):
             self.__batch_insert([], False, True)
             self.__batch_insert([], True, True)
             self._airdrop_db.commit_transaction()
+            self.__send_slack_message(f"Successfully completed updating rewards for window {self._window_id}", only_registered)
```

Comment:
can we add a slac message here for an exception too

Reply:
All exceptions are sent to slack, see annotations